lib/manager/dataset_manager.js

```diff
@@ -344,8 +352,16 @@ class DatasetManager {
 
                 rtlc.on('end', async () => {
                     try {
+                        // write data to buffer (a buffer is just a json file named after the current time, and will
+                        // contain the newest update)
+                        const now = new Date();
+                        await writeFile(`${this.storage}/feed_history/${dataset['companyName']}/${now.toISOString()}.json`, JSON.stringify(rawData), 'utf8');
+                        // now, write the new data from the buffer to 'latest.json'
+                        fs.copyFileSync(`${this.storage}/feed_history/${dataset['companyName']}/${now.toISOString()}.json`, `${this.storage}/feed_history/${dataset['companyName']}/latest.json`);
```

Comment:
A sync copy is not very efficient. A better alternative would be to asynchronously write the data to all locations. For example:
```js
await Promise.all(
   writeFile('.../feed_history/...', data),
   writeFile('.../real_time/...', data),
   writeFile('.../latest.json', data)
);
```